controllers/s3utils.go

```diff
@@ -100,9 +100,6 @@ type objectStorer interface {
 		uploadContent interface{}) error
 	verifyPVUpload(bucket string, pvKeySuffix string,
 		verifyPV corev1.PersistentVolume) error
-	downloadPVs(bucket string) (pvList []corev1.PersistentVolume, err error)
-	downloadTypedObjects(bucket string,
-		objectType reflect.Type) (interface{}, error)
```

Comment:
Why are downloadPV methods deleted? Won't VRG be needing them?

Reply:
Removed by mistake.  I already saw that while updating the unit tests.  Thanks.